src/onboarding/success/OnboardingSuccessScreen.tsx

```diff
@@ -4,14 +4,20 @@ import { Image, StyleSheet, Text, View } from 'react-native'
 import Logo, { LogoTypes } from 'src/icons/Logo'
 import { background } from 'src/images/Images'
 import { nuxNavigationOptionsNoBackButton } from 'src/navigator/Headers'
-import { navigateHome } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { goToNextOnboardingScreen, onboardingPropsSelector } from 'src/onboarding/steps'
+import useTypedSelector from 'src/redux/useSelector'
 import colors from 'src/styles/colors'
 import fontStyles from 'src/styles/fonts'
 import { Spacing } from 'src/styles/styles'
 
 function OnboardingSuccessScreen() {
+  const onboardingProps = useTypedSelector(onboardingPropsSelector)
   useEffect(() => {
-    const timeout = setTimeout(() => navigateHome(), 3000)
+    const timeout = setTimeout(
+      () => goToNextOnboardingScreen(Screens.VerificationStartScreen, onboardingProps),
```

Comment:
i find this a bit confusing. the intuitive screen parameter seems like it would be Screens.OnboardingSuccessScreen. can you explain why it's not?

Reply:
Its really the first screen in the current 'step'. I've adjusted the variable names to make this very clear.